ui/selectors/multichain.test.ts

```diff
@@ -425,22 +439,26 @@ describe('Multichain Selectors', () => {
         network: 'mainnet',
         account: MOCK_ACCOUNT_BIP122_P2WPKH,
         asset: MultichainNativeAssets.BITCOIN,
+        chainId: BtcScope.Mainnet,
       },
       {
         network: 'testnet',
         account: MOCK_ACCOUNT_BIP122_P2WPKH_TESTNET,
         asset: MultichainNativeAssets.BITCOIN_TESTNET,
+        chainId: BtcScope.Testnet,
       },
     ])(
       'returns cached balance if account is non-EVM: $network',
       ({
         account,
         asset,
+        chainId,
       }: {
         account: InternalAccount;
         asset: MultichainNativeAssets;
+        chainId: CaipChainId;
```

Comment:
And we might also need to use `SupportedCaipChaindId` here too, otherwise the compiler will also complain here (if you're ok with my suggestion above)

Reply:
https://github.com/MetaMask/metamask-extension/pull/31702/commits/b3477ebfe4f347fc374750c608f833f3a735a57c